src/grpc/proto/keybroker.proto

```diff
@@ -55,7 +55,18 @@ message SecretResponse {
     string LaunchSecretData = 2;
 }
 
+message OnlineSecretRequest {
+    string ClientId = 1;
+    repeated RequestDetails SecretRequests = 2;
+}
+
+message OnlineSecretResponse {
+    string Payload = 1;
+    string IV = 2;
+}
```

Comment:
For forward compatibility, I suggest adding `string Algorithm = 1` which will be set to `AES-GCM-256` for now.